orangecontrib/network/widgets/OWNxFile.py

```diff
@@ -81,7 +80,7 @@ def reload_data(self):
     def populate_comboboxes(self):
         self.filecombo.clear()
         for file in self.recentFiles or (NONE,):
-            self.filecombo.addItem(path.basename(file))
+            self.filecombo.addItem(path.basename(file[0]))
```

Comment:
Could you elaborate the addition of [0] here and in couple of changes below? If it is unrelated to the move to AnyQt, it might deserve a separate commit describing why it was changed.